contrib_bots/bots/virtual_fs/virtual_fs.py

```diff
@@ -38,22 +38,22 @@ def handle_message(self, message, client, state_handler):
 
 def get_help():
     return '''
-The "fs" commands implement a virtual file system for a stream.
+This bot implements a virtual file system for a stream.
 The locations of text are persisted for the lifetime of the bot
 running, and if you rename a stream, you will lose the info.
 Example commands:
 
 ```
-fs mkdir: create a directory
-fs ls: list a directory
-fs cd: change directory
-fs pwd: show current path
-fs write: write text
-fs read: read text
-fs rm: remove a file
-fs rmdir: remove a directory
+@mention-bot mkdir: create a directory
```

Comment:
No, all the bots are called by the name we fix.

http://zulip.readthedocs.io/en/latest/bots-guide.html#how-to-run-a-bot
All the bots are callable by the name we fix in 1st point above.